backend/create_admin.py

```
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+"""Script to create the first admin user"""
+
+from app.database import get_db
+from app.crud import user
+from app.schemas.user import UserCreate
+
+def create_admin_user():
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)